app/helpers/authentication_helper.rb

```diff
@@ -24,4 +24,12 @@ def recaptcha_configured_and_enabled?
       SiteConfig.recaptcha_site_key.present? &&
       SiteConfig.require_captcha_for_email_password_registration
   end
+
+  def forem_creator_flow_enabled?
+    Flipper.enabled?(:creator_onboarding) && waiting_on_first_user?
```

Comment:
Is `waiting_on_first_user?` something you can reset easily if you want to test this flow multiple times? Wondering if maybe you just want to only check if it's enabled to make testing easier.